src/main/java/graphql/schema/validation/SchemaValidationErrorType.java

```diff
@@ -3,7 +3,7 @@
 import graphql.Internal;
 
 @Internal
-public enum SchemaValidationErrorType {
+public enum SchemaValidationErrorType implements SchemaValidationErrorClassification{
```

Comment:
yeah enums are a mistake in an API - albeit this is not actually API yet but rather @Internal

Either way enums lock you into a close type system and that is bad for errors since we can always invent new errors and error types